applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -64,14 +64,25 @@ init() ->
 -spec allowed_methods() -> http_methods().
 -spec allowed_methods(path_token()) -> http_methods().
 -spec allowed_methods(path_token(), path_token()) -> http_methods().
-allowed_methods() ->
-    [?HTTP_GET, ?HTTP_PUT].
-allowed_methods(_) ->
-    [?HTTP_GET, ?HTTP_POST, ?HTTP_PATCH, ?HTTP_DELETE].
-allowed_methods(_, <<"details">>) ->
-    [?HTTP_GET];
-allowed_methods(_, <<"mute">>) ->
-    [?HTTP_POST].
+-spec allowed_methods(path_token(), path_token(), path_token()) -> http_methods().
+
+allowed_methods() -> [?HTTP_GET, ?HTTP_PUT].
+
+allowed_methods(_) -> [?HTTP_GET, ?HTTP_POST, ?HTTP_PATCH, ?HTTP_DELETE].
+
+allowed_methods(_, <<"details">>) -> [?HTTP_GET];
+
+allowed_methods(_, <<"lock">>) -> [?HTTP_POST];
```

Comment:
@jamhed Howdy, I understand that there was precedent but having actions in the REST api is not good practice and something we avoid...  We should discuss alternatives on the bridge tomorrow (or here)...

For example I would propose something such as:

Lock/Unlock Conference:
PUT v2/accounts/{account_id}/conferences/{conference_id}/lock 
DELETE v2/accounts/{account_id}/conferences/{conference_id}/lock

Kick:
DELETE v2/accounts/{account_id}/conferences/{conference_id}/participants/{participant_id

Kick All:
DELETE v2/accounts/{account_id}/conferences/{conference_id}/participants

Mute/Unmute
PUT v2/accounts/{account_id}/conferences/{conference_id}/participants/{participant_id}/mute
DELETE v2/accounts/{account_id}/conferences/{conference_id}/participants/{participant_id}/mute

Deaf/Undeaf
PUT v2/accounts/{account_id}/conferences/{conference_id}/participants/{participant_id}/deaf
DELETE v2/accounts/{account_id}/conferences/{conference_id}/participants/{participant_id}/deaf